docs-src/go/listen-to-heartbeats.md

```diff
@@ -4,7 +4,12 @@ title: Listen to Heartbeats
 description: When an Activity sends a Heartbeat, be sure that you can see the Heartbeats in your test code so that you can verify them.
 sidebar_label: Listen to Heartbeats
 tags:
-  - guide-context
+  - developer-guide-doc-type
+  - go sdk
+  - testing
+  - testing-guide-doc-type
```

Comment:
i like `how-to doc type` and `explanation doc-type` and `reference doc-type`

Not sure about these others at this point.